plugins/modules/identity/ipa/ipa_dnszone.py

```diff
@@ -32,6 +32,12 @@
     default: "false"
     choices: ["false", "true"]
     type: str
+  allowsyncptr:
+    description: Allow synchronization of forward and reverse records in the zone
+    required: false
+    default: "false"
+    choices: ["false", "true"]
```

Comment:
Should this be a string, or boolean? If this should be a boolean, please use `type: bool` and don't add `choices`.

(I do see that other options do this strange thing as well, but that really looks wrong to me.)

Reply:
Agreed - given API docs, this should be `type:bool`
> --allow-sync-ptr=BOOL
>                        Allow synchronization of forward (A, AAAA) and reverse
>                       (PTR) records in the zone

Updates made accordingly.